src/views/Correction.vue

```diff
@@ -28,9 +28,9 @@
 
     <CompletingParty class="mt-10" sectionNumber="1." />
 
-    <Detail class="mt-10" sectionNumber="2." />
+    <Detail class="mt-10" sectionNumber="2." validate="true" />
 
-    <CertifySection class="mt-10" sectionNumber="3." />
+    <CertifySection class="mt-10" sectionNumber="3." validate="true" />
```

Comment:
Again, this is to enable error styling on a correction filing, since it's a single page and it's nice to see what's missing (and thus why the File and Pay button is disabled). Approved by Linda and Yui at standup today.